protos/sources/vega/markets.proto

```diff
@@ -20,6 +20,16 @@ message AuctionDuration {
   uint64 volume = 2;
 }
 
+// Spot product definition
+message Spot {
+  // Asset ID of the underlying base asset for the spot.
+  string base_asset = 1;
+  // Asset ID of the underlying quote asset for the spot.
```

Comment:
```suggestion
  // Asset ID of the underlying base asset for the spot.
  string base_asset = 1;
  // Asset ID of the underlying quote asset for the spot.
```

Is "spot market" ok to add here? "for the spot" just seems a bit random

Reply:
spot product I think is more appropriate